apps/opik-frontend/src/components/pages-shared/traces/TraceDetailsPanel/TraceTreeViewer/TraceTreeViewer.tsx

```diff
@@ -193,9 +219,14 @@ const TraceTreeViewer: React.FunctionComponent<TraceTreeViewerProps> = ({
         <div className="sticky top-0 z-10 flex flex-row items-center justify-between gap-2 bg-background pb-2 pl-6 pr-4 pt-4">
           <div className="flex h-8 items-center gap-1">
             <div className="comet-title-xs">{title}</div>
-            <div className="comet-body-s text-muted-slate">
-              {!hasSearchOrFilter ? traceSpans.length : searchIds.size} items
-            </div>
+            <TooltipWrapper content="View all spans of this trace in table view">
+              <button
+                onClick={handleNavigateToSpans}
+                className="comet-body-s text-muted-slate underline hover:text-foreground"
+              >
+                {!hasSearchOrFilter ? traceSpans.length : searchIds.size} items
```

Comment:
i would say that you refer to spans here, not `items`, to be more precise cuz it's a bit confusing to see 2 elements, and then see 1 items label

<img width="511" height="747" alt="image" src="https://github.com/user-attachments/assets/90a5027a-df7c-4c26-8cab-2daebabf58d5" />